llvm/lib/Target/AMDGPU/BUFInstructions.td

```diff
@@ -781,7 +781,7 @@ class MUBUF_AtomicRet_Pseudo<string opName, int addrKind,
 multiclass MUBUF_Pseudo_Atomics_NO_RTN <string opName,
                                         RegisterClass vdataClass,
                                         ValueType vdataType,
-                                        bit isFP = isFloatType<vdataType>.ret> {
+                                        bit isFP = vdataType.isFP> {
```

Comment:
Fold this into its only use on the line below?

Reply:
Done